src/components/products/list/ProductsList.js

```diff
@@ -53,18 +64,48 @@ class ProductsList extends Component {
 		this.setState({ products })
```

Comment:
I think that after you added isProductCreateModalOpen to the state, now whenever you call setState() you might need to specify the name of the property you are updating.... So in here for example I think it should be changed to `this.setState({products: products})`

Reply:
just like you did in openProductModal() or closeProductModal()

Reply:
Can be for clarity, just ESLint is saying that 'Expected property shorthand.'